source/topic3.rst

```diff
@@ -768,6 +768,25 @@ Friend Count
         return friendCount;
     }
 
+.. code-block:: java
+    :linenos:
+    :emphasize-lines: 4
+
+    public String toString() {
+        String s = "";
+        for (int i = 0; i < friendCount; ++i) {
+            s = s + friends[i].toString() + "\n";
```

Comment:
`toString()` is called automatically, so it's unnecessary here. Could be information overload though, being explicit is always fine.

Reply:
I will add this point to the aside in the string builder thing.